mem/data/test/DataMemControllerRTL_test.py

```diff
@@ -209,7 +209,6 @@ def test_mem_controller(cmdline_opts):
   ]
 
   th = TestHarness(InterCgraPktType,
-                   CgraPayloadType,
                    DataType,
                    DataAddrType,

```

Comment:
I think the `DataType` and `DataAddrType` can be derived from the `InterCgraPktType`.